expressjs/src/routes/SupervisorRoutes.ts

```diff
@@ -32,8 +33,10 @@ function removeApiKeys(obj: varRemoval) {
   return obj
 }
 
-// Routes
-router.post('/supervisor', function (req, res) {
+// -- Routes -- //
+// Note that this route uses the queueCache middleware. Be aware of its implications
+// when doing development.
+router.post('/supervisor', queueCache, function (req, res) {
```

Comment:
Note, the cache only applies to routes that have the queueCache included in the ExpressJS router line such as seen here. Anything without this will not be affected at all by the middleware.

Reply:
excellent!